lua/nvim-tree/lib.lua

```diff
@@ -467,7 +467,11 @@ function M.change_dir(name)
   end
   current_tab = new_tab
 
-  vim.cmd('lcd '..vim.fn.fnameescape(foldername))
+  if vim.g.nvim_tree_change_dir_global == 1 then
+    vim.cmd('cd '..vim.fn.fnameescape(foldername))
```

Comment:
did you try this change with `opts.update_cwd` to true ? it might create bugs.

Reply:
I have `opts.update_cwd` enabled by default in my setup. I used this branch since I created the PR 27 days ago and haven't noticed any issues or unexpected behavior.